cvat/apps/engine/task.py

```diff
@@ -340,6 +340,11 @@ def _create_thread(tid, data, options):
     else:
         segments = []
     _save_task_to_db(db_task, segments)
+
+    job.meta['status'] = 'Image meta cache is being created'
+    job.save_meta()
+    make_image_meta_cache(db_task)
```

Comment:
I don't see this function defined anywhere
did you add all files to this pull request?

Reply:
this function was written by cvat authors and defined there:
https://github.com/daedaleanai/cvat/blob/62f942f4253120b00e4cdf78156a2dfd349f35aa/cvat/apps/engine/task.py#L52-L81

Reply:
Thanks! GitHub search failed me here for some reason.

Reply:
It turns out that forked repositories are not searchable, but search in the parent repository works.